.github/workflows/validation.yml

```diff
@@ -111,6 +113,13 @@ jobs:
           --nuvlaedge_version ${{ github.event.inputs.nuvlaedge_version || 'latest' }} 
           --deployment_branch ${{ github.event.inputs.deployment_branch || 'None' }}
 
+      - name: Save logs as artifacts if requested or failed
+        if: ${{ github.event.inputs.retrieve_logs }} || ${{ steps.validate.outcome }} == 'success'
```

Comment:
@ignacio-penas Following the intent, shouldn't it instead be
```
${{ steps.validate.outcome }} != 'success'
```

Reply:
Yes, you are completely right